mysqld_exporter.go

```diff
@@ -696,6 +767,77 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) {
 		}
 	}
 
+	if *perfEventsStatements {
+		// Timers here are returned in picoseconds.
+		perfSchemaEventsStatementsRows, err := db.Query(perfSchemaEventsStatementsQuery + strconv.Itoa(*perfEventsStatementsLimit))
```

Comment:
I suggest putting this into a `fmt.Sprintf`. Otherwise changing the statement text above will break down here.